lib/network/modules/components/Edge.js

```diff
@@ -543,6 +545,26 @@ class Edge {
       return;
     }
 
+    // get the via node from the edge type
+    let viaNode = this.edgeType.getViaNode();
```

Comment:
If it can be const, make it const, thanks.

Reply:
NP, I will do.